hermes-mock/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'groovy'
+    id 'java'
+}
+
+dependencies {
+    compile group: 'junit', name: 'junit', version: '4.11'
+    compile group: 'com.github.tomakehurst', name: 'wiremock', version: '2.14.0'
+    compile group: 'com.jayway.awaitility', name: 'awaitility', version: '1.6.1'
+    compile group: 'org.apache.avro', name: 'avro', version: '1.8.1'
```

Comment:
I think it's the right moment to use shared and well defined versions in `{project root}/build.gradle`, otherwise the versions will start to diverge